content/source/docs/cloud/cost-estimation/index.html.md

```diff
@@ -7,7 +7,7 @@ page_title: "Overview - Cost Estimation - Terraform Cloud and Terraform Enterpri
 
 -> **Note:** Cost estimation is a paid feature, available as part of the **Team & Governance** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
 
-> For a hands-on tutorial, try the [Verify Costs in Policies](https://learn.hashicorp.com/terraform/cloud-getting-started/cost-estimation?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) guide on HashiCorp Learn.
+> **Hands-on:** Try the [Verify Costs in Policies](https://learn.hashicorp.com/tutorials/terraform/cost-estimation?in=terraform/cloud-get-started&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
```

Comment:
```suggestion
> **Hands-on:** Try the [Control Costs with Policies](https://learn.hashicorp.com/tutorials/terraform/cost-estimation?in=terraform/cloud-get-started&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
```

The name changed :o 
![image](https://user-images.githubusercontent.com/7556325/94887535-6b958d00-042b-11eb-8075-a10f43390281.png)